docs/crr-to-aws-s3.md

```diff
@@ -75,3 +75,334 @@ env_replication_endpoints:
 
 * Backbeat updates the source object's metadata to store AWS' version id and set
   replication status as `COMPLETED`
+
+## Setting up Cross-Region Replication (CRR) to AWS
+
+This section contains steps to set up cross-region replication from Scality S3
+to Amazon Web Services S3 using the Backbeat replication engine. Two subsections
+are provided with different methods: the subsection "Basic Example" uses a
+built-in command line tool, while the subsection "Detailed Example" takes a more
+manual approach. Lastly, the subsection "Test Object Replication" offers a way
+to confirm that replication is setup properly.
+
+### Basic Example
+
+1. Create the target bucket:
+
+    ```
+    aws s3api create-bucket \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+2. Enable versioning on the target bucket:
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <target-bucket> \
+    --versioning-configuration Status=Enabled \
+    --profile <target-profile>
+    ```
+
+3. Generate account credentials for the source site using the command:
+
+    ```
+    ansible-playbook -i env/client-template/inventory \
+    tooling-playbooks/generate-account-accesskey.yml
+    ```
+
+4. At the source, log in to the docker shell of a Backbeat container:
+
+    ```
+    docker exec -it scality-backbeat-1 bash
```

Comment:
On light-local it appears as `scality-backbeat-127.0.0.2-1`, not sure on a full stack deployment but it could be the same.

Reply:
On a full stack deployment the name of the docker container is `scality-backbeat-1`. I will add a note here about the difference.